apps/web/src/pages/templates/workflow/DigestMetadata.tsx

```diff
@@ -11,6 +12,10 @@ import { RegularInfo } from './digest/icons/RegularInfo';
 import { TimedDigestMetadata } from './TimedDigestMetadata';
 import { RegularDigestMetadata } from './RegularDigestMetadata';
 
+const GroupStyled = styled(Group)`
+  gap: 18px;
+`;
```

Comment:
this is the accordion header group for the timed digest, I've made a gap between the image and text a little smaller so the text will be contained in the header and won't break it